aws/resource_aws_api_gateway_usage_plan.go

```diff
@@ -355,7 +355,7 @@ func resourceAwsApiGatewayUsagePlanUpdate(d *schema.ResourceData, meta interface
 				operations = append(operations, &apigateway.PatchOperation{
 					Op:    aws.String("replace"),
 					Path:  aws.String("/throttle/rateLimit"),
-					Value: aws.String(strconv.Itoa(d["rate_limit"].(int))),
+					Value: aws.String(strconv.FormatFloat(d["rate_limit"].(float64), 'E', -1, 64)),
```

Comment:
Maybe a nitpick, but is there any value in exposing the exponent? i.e. in using `E`? Is the API actually going to parse it correctly and do something with it?

Reply:
Probably no value, preferred the use of the `f` format instead